src/dftbp/poisson/parameters.F90

```diff
@@ -236,6 +237,20 @@ subroutine set_contdir(dir)
     end if
   end subroutine set_contdir
 
+  !> Set labels for contacts
+  subroutine set_contlabels(names)
+
+    !> Names of contacts
+    character(mc), intent(in) :: names(:)
+
+    if (allocated(contnames)) then
+      deallocate(contnames)
+    end if
+    allocate(contnames(size(names)))
+    contnames(:) = names
```

Comment:
Automatic LHS allocation does not work in this context, *i.e.* is buggy in our compilers?

Reply:
Now changed to automatic.